packages/video_player/video_player_web/lib/src/video_player.dart

```diff
@@ -249,17 +246,17 @@ class VideoPlayer {
     _videoElement.removeAttribute('controlsList');
     _videoElement.removeAttribute('disablePictureInPicture');
     if (_onContextMenu != null) {
-      _videoElement.removeEventListener('contextmenu', _onContextMenu);
+      _videoElement.removeEventListener('contextmenu', _onContextMenu!.toJS);
```

Comment:
I think `.toJS` returns new function, and therefore `removeEventListener` won't work

Reply:
Good catch!